src/components/Ticket/utils/useGetTicketData.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useLocation } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import { getTicketByPnr } from '../../../api/endpoints/ticket.api';
+import { ITicket } from '../../../types';
+
+export const useGetTicketData = (pnrNumber: string | undefined | null) => {
```

Comment:
pnrNumber does not seem to be a null or undefined..

Reply:
Resolved in 3f93f732392e266c69cb321b4b77c8f9c7cc9642